core/src/main/java/io/aiven/klaw/config/ManageDatabase.java

```diff
@@ -422,6 +422,10 @@ public String getKwPropertyValue(String kwKey, int tenantId) {
     }
   }
 
+  public void addUserToCache(UserInfo newUser) {
```

Comment:
so we can delete this method ?

Reply:
yes! damn I knew I had to do that and I never included it in the commit!